src/shogun/regression/LeastAngleRegression.cpp

```diff
@@ -123,12 +120,13 @@ bool CLeastAngleRegression::train_machine(CFeatures* data)
 	else
 		SG_ERROR("Feature-type (%d) must be of type F_SHORTREAL (%d), F_DREAL (%d) or F_LONGREAL (%d).\n", 
 			data->get_feature_type(), F_SHORTREAL, F_DREAL, F_LONGREAL)
+
+	return false;
```

Comment:
`false` is good... better to be safe than sorry :P and anyhow it's almost as if you put it into `else` branch...